src/main/java/hudson/plugins/release/SafeParametersAction.java

```diff
@@ -0,0 +1,69 @@
+package hudson.plugins.release;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+import hudson.EnvVars;
+import hudson.Extension;
+import hudson.model.EnvironmentContributor;
+import hudson.model.ParameterValue;
+import hudson.model.ParametersAction;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+
+@Restricted(NoExternalUse.class)
+public class SafeParametersAction extends ParametersAction {
```

Comment:
I would not do it public at all. Restricted does not protect from misusages in Pipelines and system groovy scripts